ui/de-lib/src/main/java/org/iplantc/de/client/services/SearchServiceFacade.java

```diff
@@ -18,7 +18,7 @@
  * <h2>Endpoints Used</h2>
  * <ul>
  * <li><a href=
- * "https://github.com/iPlantCollaborativeOpenSource/Donkey/blob/dev/doc/endpoints/filesystem/search.md#endpoints"
+ * "https://github.com/iPlantCollaborativeOpenSource/Terrain/blob/dev/doc/endpoints/filesystem/search.md#endpoints"
```

Comment:
These URLs should become https://github.com/iPlantCollaborativeOpenSource/DE/blob/master/services/terrain/doc/endpoints/...